<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestDual</class>
 <widget class="QMainWindow" name="TestDual">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test Dual</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../data/logo.qrc">
    <normaloff>:/vlc-qt/icons/48x48/vlc-qt.png</normaloff>:/vlc-qt/icons/48x48/vlc-qt.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="group1">
      <property name="title">
       <string>Instance 1</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="VlcVideoWidget" name="video1" native="true"/>
       </item>
       <item row="4" column="0">
        <widget class="VlcVolumeSlider" name="volume1" native="true"/>
       </item>
       <item row="5" column="0">
        <widget class="VlcSeekWidget" name="seek1" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pause1">
         <property name="text">
          <string>Pause 1</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="stop1">
         <property name="text">
          <string>Stop 1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="open1">
         <property name="text">
          <string>Open 1</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="group2">
      <property name="title">
       <string>Instance 2</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="VlcVideoWidget" name="video2" native="true"/>
       </item>
       <item row="4" column="0">
        <widget class="VlcVolumeSlider" name="volume2" native="true"/>
       </item>
       <item row="5" column="0">
        <widget class="VlcSeekWidget" name="seek2" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pause2">
         <property name="text">
          <string>Pause 2</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="stop2">
         <property name="text">
          <string>Stop 2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="open2">
         <property name="text">
          <string>Open 2</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VlcVideoWidget</class>
   <extends>QWidget</extends>
   <header>../src/gui/VideoWidget.h</header>
   <container>1</container>
   <slots>
    <signal>fullscreen()</signal>
    <slot>toggleFullscreen()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>VlcVolumeSlider</class>
   <extends>QWidget</extends>
   <header>../src/gui/VolumeSlider.h</header>
   <container>1</container>
   <slots>
    <slot>mute()</slot>
    <slot>volumeUp()</slot>
    <slot>volumeDown()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>VlcSeekWidget</class>
   <extends>QWidget</extends>
   <header>../src/gui/SeekWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../data/logo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>TestDual</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
